tic_tac_toe.py

```diff
@@ -0,0 +1,111 @@
+import random
+
+
+class TicTacToe:
+
+    def __init__(self):
+        self.board = [None]*9
+
+    def make_move(self, row: int, col: int, player):
+        position = col*3 + row
```

Comment:
According to the board defined in function `display_board`, it should be:
```position = row*3 + col```

Please also add a comment here to explain what is `position`, any why is it designed like this.